### PR TITLE
Updates Kotlin to 1.0.4, latest Android Gradle plugin, and targets API 25

### PR DESCRIPTION
**Pull request by rafaeltoledo**

### CODE REVIEW

**Comment by antoniolg** [comment]
Sorry! I merged other PRs and this now has conflicts. Could you please resolve it? Thanks for your help.

**Comment by rafaeltoledo** [comment]
Just fixed the conflicts and now, as released yesterday, this PR now targets API 25 instead of 24 :)

File: app/build.gradle [hunk 1]
```diff
@@ -27,7 +27,7 @@ android {
         versionCode parent.ext.androidVersionCode
         versionName parent.ext.androidVersionName
 
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner 'android.support.test.runner.AndroidJUnitRunner'
```
Comment (antoniolg) [inline app/build.gradle:30]: I don't very much like the incongruence added between `'` and `"`. We need `"` for the strings that require a variable, so why not leaving all the strings like that? Both are equivalent.

**Review by antoniolg — CHANGES_REQUESTED** [review]
Apart from that little nitpick, everything looks OK. Could you please revert to the use of `"`? Thanks, and sorry!

**Comment by antoniolg** [comment]
This PR is obsolete, closing it.

**Pull request closed without merging**